sirepo/reply.py

```diff
@@ -396,6 +406,9 @@ def render_static_jinja(self, base, ext, j2_ctx, cache_ok=False):
             return r.headers_for_cache(path=p)
         return r.headers_for_no_cache()
 
+    def set_cookies_for_deletion(self, names_and_paths):
```

Comment:
delete_third_party_cookies is a better name. It should check that it isn't our cookie name (sirepo.cookie can export the name).

Reply:
https://github.com/radiasoft/sirepo/pull/5523/commits/711e56e4b6d7aae82470e59ce8f2c1e71199f412